lib/Backend/arm/EncoderMD.cpp

```diff
@@ -2241,7 +2241,7 @@ EncoderMD::BaseAndOffsetFromSym(IR::SymOpnd *symOpnd, RegNum *pBaseReg, int32 *p
         if (func->GetMaxInlineeArgOutSize() != 0)
         {
             Assert(func->HasInlinee());
-            Assert(baseReg == RegSP);
+            Assert(baseReg == func->HasTry() ?  RegR7 : RegSP);
```

Comment:
I believe you need the parentheses here, otherwise this is interpreted as `(baseReg == func->HasTry()) ? RegR7 : RegSP`

Reply:
ah, yes! fixed.